.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -76,8 +87,8 @@ This is a checklist for PR authors & reviewers. Please make sure to complete all
 - [ ] I verified testing steps are clear and they cover the changes made in this PR
     - [ ] I verified the steps for local testing are in the `Tests` section
     - [ ] I verified the steps for Staging and/or Production testing are in the `QA steps` section
-    - [ ] I verified the steps cover failure scenarios (if applicable, i.e. verify an input displays the correct error message if the entered data is not correct)
-    - [ ] I verified steps cover offline scenarios (if applicable, i.e. verify the default avatar icon is displayed if app is offline)
+    - [ ] I verified the steps cover any possible failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
+    - [ ] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
```

Comment:
Is the offline behaviour always going to match the expected behaviour? do we need a new section in the GH issue to describe how it should work when offline if it is different?